src/snowcli/cli/snowpark_shared.py

```diff
@@ -81,31 +80,33 @@ def snowpark_package(
             )
             if do_download:
                 log.info("Installing non-Anaconda packages...")
-                should_pack, second_chance_results = utils.install_packages(
-                    "requirements.other.txt",
+                should_continue, second_chance_results = utils.install_packages(
+                    REQUIREMENTS_OTHER,
                     check_anaconda_for_pypi_deps,
                     package_native_libraries,
                 )
-                if should_pack:
-                    # add the Anaconda packages discovered as dependencies
-                    if second_chance_results is not None:
-                        split_requirements.snowflake = (
-                            split_requirements.snowflake
-                            + second_chance_results.snowflake
-                        )
+                # add the Anaconda packages discovered as dependencies
+                if should_continue and second_chance_results:
+                    split_requirements.snowflake = (
+                        split_requirements.snowflake + second_chance_results.snowflake
+                    )
 
         # write requirements.snowflake.txt file
         if split_requirements.snowflake:
-            log.info("Writing requirements.snowflake.txt file...")
-            with open(
-                "requirements.snowflake.txt",
-                "w",
-                encoding="utf-8",
-            ) as f:
-                for package in utils.deduplicate_and_sort_reqs(
-                    split_requirements.snowflake
-                ):
-                    f.write(package.line + "\n")
+            _write_requirements_file(
+                REQUIREMENTS_SNOWFLAKE,
+                utils.deduplicate_and_sort_reqs(split_requirements.snowflake),
+            )
 
     zip_dir(source=source, dest_zip=artefact_file)
-    log.info("Deployment package now ready: app.zip")
+
+    if os.path.exists(".packages"):
+        zip_dir(source=Path(".packages"), dest_zip=artefact_file, mode="a")
+    log.info(f"Deployment package now ready: %s", artefact_file)
+
+
+def _write_requirements_file(file_name: str, reqirements: List[Requirement]):
+    log.info(f"Writing %s file", file_name)
+    with open(file_name, "w", encoding="utf-8") as f:
+        for package in reqirements:
+            f.write(package.line + "\n")
```

Comment:
Done, but had to add another list comprehension, as join doesn`t convert Requirements to string

Reply:
So maybe the for loop was a batter solution instead of nesting comprehentions
```python
for package in requirements:
    f.write(f"{package.line}\n")
```

Reply:
Done